dashboard/app/controllers/puzzle_ratings_controller.rb

```diff
@@ -0,0 +1,15 @@
+class PuzzleRatingsController < ApplicationController
+  def create
+    status_code = :unauthorized
+    if PuzzleRating.enabled?
+      valid_request = PuzzleRating.create(
```

Comment:
create will return false if invalid, so you could just use the return value of create and skip the valid? call